end2end-test-examples/spanner/src/main/java/io/grpc/spanner/SpannerClient.java

```diff
@@ -154,7 +153,12 @@ public void singleQuery() {
     ResultSet rs =
         dbClient
             .singleUse()
-            .executeQuery(Statement.of("SELECT Key,StringValue FROM " + TABLE_NAME));
+            .executeQuery(
+                Statement.of(
+                    "SELECT Key,StringValue FROM "
+                        + TABLE_NAME
+                        + " WHERE 123  != "
+                        + uniqueString()));
```

Comment:
It's ok if it works like this and gets sent to the backend. But can `seq` be equal to 123 at some point? In this case the backend won't return a record back. It's not a problem for our test but if we will analyze responses it may puzzle us in the future.

Reply:
`seq` is always 8 digits, like `00000123`. From log, `00000123` can also get a Status OK.

Reply:
Yes, the status will be OK, but the response might not have the record returned (thus much fewer bytes than other responses). We can change it to ` WHERE -123 != ` or use the column.

Reply:
Done